features/git-sync-all/feature_branch_merge_main_branch_conflict/with_tracking_branch_updates/first_branch/with_open_changes.feature

```diff
@@ -0,0 +1,135 @@
+      | feature2 | local and remote | Merge branch 'main' into feature2 |                      |
+      |          |                  | main commit                       | conflicting_file     |
+      |          |                  | feature2 commit                   | feature2_file        |
+
+
+  @finishes-with-non-empty-stash
+  Scenario: continuing without resolving conflicts
+    When I run `git sync-all --continue` while allowing errors
+    Then it runs no Git commands
+    And I get the error "You must resolve the conflicts before continuing the git sync"
+    And I am still on the "feature1" branch
+    And I don't have an uncommitted file with name: "uncommitted"
+    And my repo still has a merge in progress
+
+
+  Scenario: continuing after resolving conflicts
+    Given I resolve the conflict in "conflicting_file"
+    And I run `git sync-all --continue`
+    Then it runs the Git commands
+      | BRANCH   | COMMAND                             |
+      | feature1 | git commit --no-edit                |
+      | feature1 | git push                            |
+      | feature1 | git checkout feature2               |
+      | feature2 | git merge --no-edit origin/feature2 |
+      | feature2 | git merge --no-edit main            |
+      | feature2 | git push                            |
+      | feature2 | git checkout main                   |
+      | main     | git stash pop                       |
+    And I end up on the "main" branch
+    And I again have an uncommitted file with name: "uncommitted" and content: "stuff"
+    And I have the following commits
+      | branch   | location         | message                                                      | files                |
+      | main     | local and remote | main commit                                                  | conflicting_file     |
+      | feature1 | local and remote | Merge branch 'main' into feature1                            |                      |
+      |          |                  | main commit                                                  | conflicting_file     |
+      |          |                  | Merge remote-tracking branch 'origin/feature1' into feature1 |                      |
+      |          |                  | feature1 local commit                                        | conflicting_file     |
+      |          |                  | feature1 remote commit                                       | feature1_remote_file |
+      | feature2 | local and remote | Merge branch 'main' into feature2                            |                      |
+      |          |                  | main commit                                                  | conflicting_file     |
+      |          |                  | feature2 commit                                              | feature2_file        |
+
+
+  Scenario: continuing after resolving conflicts and committing
```

Comment:
In other specs we also tested the scenario to continue without committing here. If you only want to check one, I think that one is a bit more inclusive than the one with committing.